.ci/test-spelling-unknown-words.sh

```diff
@@ -9,11 +9,12 @@ spellchecker='config/jsoref-spellchecker'
 temp='.ci-temp'
 whitelist_path="$spellchecker/whitelist.words"
 dict="$temp/english.words"
-word_splitter="$spellchecker/spelling-unknown-word-splitter.pl"
-run_output="$spellchecker/unknown.words"
+word_splitter="$temp/spelling-unknown-word-splitter.pl"
+run_output="$temp/unknown.words"
+
+mkdir -p $temp
```

Comment:
This was moved here as all files need it. There is no file existence check for `$run_output`, so it would always need to be run regardless.